static/app/components/events/eventStatisticalDetector/eventDifferentialFlamegraph.tsx

```diff
@@ -82,7 +82,7 @@ export function EventDifferenialFlamegraph(props: EventDifferenialFlamegraphProp
           <EventDifferentialFlamegraphView before={before} after={after} />
         </FlamegraphStateProvider>
       </FlamegraphThemeProvider>
-    </div>
+    </Fragment>
```

Comment:
This `Fragment` looks redundant

Reply:
It looks like it yeah, but it's required. The children prop indentation makes it look off